sphinx/writers/latex.py

```diff
@@ -813,7 +813,7 @@ def depart_desc_content(self, node: Element) -> None:
         pass
 
     def visit_seealso(self, node: Element) -> None:
-        self.body.append('\n\n\\sphinxstrong{%s:}\n\n' % admonitionlabels['seealso'])
+        self.body.append('\n\n\\sphinxstrong{%s:}\n\\nopagebreak\n\n' % admonitionlabels['seealso'])
```

Comment:
>./sphinx/writers/latex.py:816:96: E501 line too long (100 > 95 characters)
>https://github.com/sphinx-doc/sphinx/pull/8519/checks?check_run_id=1500490152

Could you fold this line, please?

Reply:
@tk0miya I'm actually for the 79 char line length limit more than most people, but in this case IMHO it is irrelevant as the rest of the file has many exceptions (in many cases much more lengthier than this). Keeping it on one line made the diff more clear as it only contains the relevant change.

Reply:
Would you like to discuss the coding rule of this project to merge this change? If so, please file another issue first.

Reply:
No, I will not insist if the coding guidelines are not flexible.

Reply:
Thank you for update. I'll merge this soon after tests and linting passed.